services/game-session-service/src/main/resources/db/migration/V8__remote_followup_target_instance_effect_identity.sql

```
@@ -0,0 +1,4 @@
+DROP INDEX IF EXISTS idx_remote_followup_target_region_epoch_effect;
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)